README.md

```diff
@@ -76,7 +76,7 @@ resolvers := Seq(
 ```
 libraryDependencies ++= Seq(
   ...
-  "com.socrata" % "balboa-client" % "0.13.0",
+  "com.socrata" % "balboa-client_2.10" % "0.14.0",
```

Comment:
It might be a good idea to start integrating krah's sbt socrata style plugins at a future date. It might require a bit cleanup.